docs/applications/messaging/installing-prosody-xmpp-server-on-ubuntu-10-04-lucid.md

```diff
@@ -4,32 +4,32 @@ author:
   name: Linode
   email: docs@linode.com
 description: 'Installation and basic usage guide for Prosody, a lightweight XMPP server on Ubuntu 10.04 (Lucid).'
-keywords: 'prosody,prosody ubuntu lucid,prosody.im,xmpp,real time messaging,lua'
+keywords: ["prosody", "prosody ubuntu lucid", "prosody.im", "xmpp", "real time messaging", "lua"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
-alias: ['communications/xmpp/prosody/ubuntu-10-04-lucid/']
-modified: Tuesday, September 24th, 2013
+aliases: ['communications/xmpp/prosody/ubuntu-10-04-lucid/']
+modified: 2013-09-24
 modified_by:
   name: Linode
-published: 'Monday, December 6th, 2010'
+published: 2010-12-06
 title: 'Installing Prosody XMPP Server on Ubuntu 10.04 (Lucid)'
 ---
 
 
 
-Prosody is a XMPP/Jabber server programmed in Lua that is simple and lightweight. Prosody uses fewer resources than its counterparts and is designed to be easy to configure and run. [ejabberd](/docs/applications/messaging/instant-messaging-services-with-ejabberd-on-ubuntu-12-04-precise-pangolin) or [OpenFire](/docs/applications/messaging/instant-messaging-services-with-openfire-on-ubuntu-12-04-lts-precise-pangolin) may be better suited for larger applications, but for most independent and small scale uses Prosody is a more resource-efficient solution. Prosody is a very good candidate for running an XMPP server for a very small base of users, or for XMPP development.
+Prosody is a XMPP/Jabber server programmed in Lua that is simple and lightweight. Prosody uses fewer resources than its counterparts and is designed to be easy to configure and run. [ejabberd](/content/applications/messaging/instant-messaging-services-with-ejabberd-on-ubuntu-12-04-precise-pangolin) or [OpenFire](/content/applications/messaging/instant-messaging-services-with-openfire-on-ubuntu-12-04-lts-precise-pangolin) may be better suited for larger applications, but for most independent and small scale uses Prosody is a more resource-efficient solution. Prosody is a very good candidate for running an XMPP server for a very small base of users, or for XMPP development.
 
-Before we begin with the installation and configuration of Prosody, we assume that you have a running and up to date installation of Ubuntu 10.04 (Lucid), have completed our [getting started](/docs/getting-started/) guide, and have logged in via SSH as root.
+Before we begin with the installation and configuration of Prosody, we assume that you have a running and up to date installation of Ubuntu 10.04 (Lucid), have completed our [getting started](/content/getting-started/) guide, and have logged in via SSH as root.
 
 Adding Software Repositories
 ----------------------------
```

Comment:
This style formatting makes it difficult to know which header is being used. Deprecate in favor of `#, ##, ###` etc

Reply:
@Guaris Can you fix?

Reply:
no prob